packages/lime-full/Makefile

```diff
@@ -23,7 +23,7 @@ define Package/$(PKG_NAME)
   SUBMENU:=Collections
   MAINTAINER:=Pau Escrich <p4u@dabax.net>
   URL:=http://libremesh.org
-  DEPENDS:=+lime-basic +lime-debug +luci-mod-admin-full +lime-map-agent +lime-docs
+  DEPENDS:=+lime-basic +lime-debug +luci-mod-admin-full +lime-map-agent +lime-docs +lime-app
```

Comment:
Shouldn't we remove luci-mod-admin-full as a dependence and add it to the full flavor?